core/src/components/datetime-button/test/overlays/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Datetime Button - Overlays</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0" />
+    <link href="../../../../../css/ionic.bundle.css" rel="stylesheet" />
+    <link href="../../../../../scripts/testing/styles.css" rel="stylesheet" />
+    <script src="../../../../../scripts/testing/scripts.js"></script>
+    <script type="module" src="../../../../../dist/ionic/ionic.esm.js"></script>
+    <style>
+      .grid {
+        display: grid;
+        grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+        grid-row-gap: 20px;
+        grid-column-gap: 20px;
+      }
+      h2 {
+        font-size: 12px;
+        font-weight: normal;
+
+        color: #6f7378;
+
+        margin-top: 10px;
+        margin-left: 5px;
+      }
+    </style>
+  </head>
+  <body>
+    <ion-app>
+      <ion-header>
+        <ion-toolbar>
+          <ion-title>Datetime Button - Overlays</ion-title>
+        </ion-toolbar>
+      </ion-header>
+      <ion-content>
+        <div class="grid">
+          <div class="grid-item">
+            <h2>Popover - Default</h2>
+
+            <ion-item>
+              <ion-datetime-button
+                slot="end"
+                id="popover-default-button"
+                datetime="popover-default-datetime"
+              ></ion-datetime-button>
+            </ion-item>
+
+            <ion-popover arrow="false">
+              <ion-datetime locale="en-US" id="popover-default-datetime" value="2022-03-15T00:43:00"></ion-datetime>
+            </ion-popover>
+          </div>
+          <div class="grid-item">
+            <h2>Modal - Default</h2>
+
+            <ion-item>
+              <ion-datetime-button
+                slot="end"
+                id="modal-default-button"
+                datetime="modal-default-datetime"
+              ></ion-datetime-button>
+            </ion-item>
+
+            <ion-modal>
+              <ion-datetime locale="en-US" id="modal-default-datetime" value="2022-03-15T00:43:00"></ion-datetime>
+            </ion-modal>
+          </div>
+          <div class="grid-item">
+            <h2>Popover - Custom</h2>
```

Comment:
For both the popover and modal, when the `md` header is present, the overlay doesn't get sized properly: https://i.gyazo.com/9b23834ac9288c72c28b87db7b370649.png